docs/esp32/quickref.rst

```diff
@@ -507,6 +507,17 @@ The DHT driver is implemented in software and works on all pins::
     d.temperature() # eg. 23.6 (°C)
     d.humidity()    # eg. 41.3 (% RH)
 
+WDT (Watchdog timer)
```

Comment:
I think this section would be better a bit higher, because it's part of the machine module it should go with other machine module things, eg after RTC.

Reply:
@dpgeorge Done. Moved higher up.